src/GitHub.TeamFoundation.14/Connect/GitHubConnectSection.cs

```diff
@@ -249,6 +291,23 @@ void OnPropertyChange(object sender, PropertyChangedEventArgs e)
                 settings.IsExpanded = IsExpanded;
         }
 
+        void InvitationSectionPropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == "IsVisible")
```

Comment:
`nameof` maybe?

Reply:
Yeah I was trying to stick with the style of the previous method which didn't use `nameof`, but you're right. I've converted this method and the previous to use `nameof`.